packages/site/src/data/vue/podcasts.ts

```diff
@@ -1,5 +1,87 @@
 import { Podcast } from "@framework/system/src/models/podcast"
 
-export const podcastTags = [] as const
+export const podcastTags = ["general", "graphQL", "vuejs", "quasar"] as const
 
-export const podcasts: Podcast<typeof podcastTags[number]>[] = []
+export const podcasts: Podcast<typeof podcastTags[number]>[] = [
+	{
+		title: "Modern Web",
+		image:
+			"https://pbcdn1.podbean.com/imglogo/image-logo/984467/modern_web_9bpnd.jpg",
+		hosts: ["ThisDot Labs"],
+		description:
+			"Modern Web is a podcast that explores next generation frameworks, standards, and techniques.",
+		rss: "https://www.podbean.com/site/podcatcher/index/blog/7YqKYcoGcvP",
+		href: "https://modernweb.podbean.com/",
+		tags: ["general"],
+	},
+	{
+		title: "Views on Vue",
+		image:
+			"https://assets.fireside.fm/file/fireside-images/podcasts/images/4/4b532b8f-dc99-4b48-90f1-023bd6c0cd90/cover_medium.jpg?v=0",
+		hosts: [
+			"Lindsay Wardell",
+			"Luke Diebold",
+			"Solomon Eseme",
+			"Steve Edwards",
+			"Charles Wood",
+		],
+		description:
+			"A weekly discussion among Vue developers about Vue and it's ecosystem.",
+		rss: "",
```

Comment:
All podcasts should have an RSS feed, for example for this one it is https://viewsonvue.com/rss

All of the blank RSS fields should be filled out, in general empty strings are never valid for required fields in framework.dev

Reply:
Removed one podcast that didn't have any RSS feed